NOTICE

```diff
@@ -1,5 +1,6 @@
 Druid - a distributed column store.
 Copyright 2012-2015 Metamarkets Group Inc.
+Copyright 2015 Yahoo! Inc.
```

Comment:
I am not a lawyer, but it might be confusing to include several copyright notices without context. I had a look at how it is done by a couple ASF projects, and it might be clearer to include the following snippet (based on the [Apache Derby notice](https://github.com/apache/derby/blob/trunk/NOTICE#L19)).

```
Portions of this software were originally developed by
Yahoo! Inc and are licensed to Metamarkets Group Inc. under the
"Metamarkets Corporate Contributor License Agreement"

The following copyright notice(s) were affixed to portions of the code
with which this file is now or was at one time distributed
and are placed here unaltered.

Copyright 2015 Yahoo! Inc.
```